webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/controllers/VolumeTracingController.scala

```diff
@@ -355,6 +365,24 @@ class VolumeTracingController @Inject()(
     }
   }
 
+  def zarrSource(token: Option[String], tracingId: String): Action[AnyContent] = Action.async { implicit request =>
+    accessTokenService.validateAccess(UserAccessRequest.readTracing(tracingId), urlOrHeaderToken(token, request)) {
+      for {
+        tracing <- tracingService.find(tracingId) ?~> Messages("tracing.notFound") ~> 404
+
+        zarrLayer = ZarrSegmentationLayer(
+          name = tracingId,
+          largestSegmentId = if (tracing.segments.nonEmpty) tracing.segments.maxBy(_.segmentId).segmentId else 1,
```

Comment:
volume tracings have a largestSegmentId field, not all have the segment list. please use that :)